src/pages/Order.js

```diff
@@ -1,12 +1,17 @@
 import OrderItem from "../components/OrderItem";
+
 import "./Order.css";
 
-export default function Order({ menuItems, onDeleteButton: deleteItem }) {
+export default function Order({ menu, order, onDeleteButton: deleteItem }) {
```

Comment:
here as well. onDeleteButton would be enough.
also the naming is not 100% correct. this article helped me a lot:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
thanks! onDeleteButton is indeed enough but I destructured the name and it's just easier to directly read what I actually did. Example: `onDeletebutton={deleteItem}`
[I read this article](https://davidwalsh.name/destructuring-alias) 😉